courses-app/src/app/auth/store/auth.actions.ts

```diff
@@ -0,0 +1,109 @@
+import {Action} from '@ngrx/store';
+import {RequestCurrentUserFail} from "../../user/store/user.actions";
+
+export const REQUEST_LOGIN_START = '[Auth] Request Login Start';
```

Comment:
Is there a reason you're using this verbose syntax of creating actions, and not import { createAction, props } from '@ngrx/store'; API ?